docbook/src/docbkx/autoscale-gettingstarted.xml

```diff
@@ -619,6 +619,76 @@ xmlns:html="http://www.w3.org/1999/xhtml" xml:id="autoscale-getting-started" xml
                     </listitem>
                 </itemizedlist>
             </para>
+<<<<<<< HEAD
```

Comment:
This extra line was left in from resolving the merge conflict I think.  It's causing errors building the docs with maven.